playbooks/hc-ansible-deployment/tasks/add_hosts_storage_domains.yml

```diff
@@ -47,3 +47,16 @@
     loop_control:
       loop_var: sd
     when: "'gluster' in groups"
+
+  - name: Create oVirt gluster network
+    ignore_errors: true
+    ovirt_network:
+      data_center: "{{ datacenter_name }}"
+      name: gluster_net
+      vm_network: false
+      auth: "{{ ovirt_auth }}"
+      clusters:
+        - name: "{{ cluster_name }}"
+          assigned: true
+          required: false
+          gluster: true
```

Comment:
This network is also a migration network, please add migration: true

Reply:
We need to ignore this patch as we decided to create a separate playbook for gluster_network creation, as the hosts are added to the cluster in the async fashion, creation of network had to wait for long time which also prolongs the deployment